openshift-ansible.spec

```diff
@@ -63,7 +63,6 @@ Summary:       Openshift and Atomic Enterprise Ansible Test Playbooks
 Requires:      %{name} = %{version}-%{release}
 Requires:      ansible >= 2.9.5
 Requires:      openssh-clients
-Requires:      python2-boto
```

Comment:
This is unrelated to the GCP script, and is already covered by #12399.  I'd suggest just limiting this PR to the removal of the GCP script.